app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/landing/LandingViewModel.kt

```diff
@@ -76,6 +77,10 @@ class LandingViewModel @Inject constructor(
                 )
             }
             .launchIn(viewModelScope)
+
+        authRepository.userStateFlow.onEach { userState ->
+            userState?.activeAccount?.let(this::handleActiveAccountChange)
```

Comment:
We usually map these type of updates to internal actions and pass them through `handleAction`.

```
authRepository
    .userStateFlow
    .map { LandingAction.Internal.ActiveAccountChangeReceive(it) }
    .onEach(::handleAction)
    .launchIn(viewModelScope)
```